Popular_Models_Guide/Llama2/trtllm_guide.md

```diff
@@ -62,35 +66,29 @@ TensorRT-LLM requires each model to be compiled for the configuration you need b
     cp /opt/tritonserver/backends/tensorrtllm/* /usr/local/lib/python3.10/dist-packages/tensorrt_llm/libs/
     ```
 
-2.  Log in to huggingface-cli
-
-    ```bash
-    huggingface-cli login --token hf_*****
-    ```
-
-3.  Compile model engines
+2.  Compile model engines
 
     The script to build Llama models is located in [TensorRT-LLM repository](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples). We use the one located in the docker container as `/tensorrtllm_backend/tensorrt_llm/examples/llama/build.py`.
     This command compiles the model with inflight batching and 1 GPU. To run with more GPUs, you will need to change the build command to use `--world_size X`.
     More details for the scripting please see the documentation for the Llama example [here](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/llama/README.md).
 
     ```bash
-    python build.py --model_dir /<path to your llama repo>/Llama-2-7b-hf/ \
+    python build.py --model_dir /Llama-2-7b-hf/ \
```

Comment:
May want to spell out the full path to build.py here instead of above for clarity in case someone copies the code-blocks before fully reading. Much like you do in the optional step below.